app/src/App.tsx

```diff
@@ -4,7 +4,27 @@ import ThingComponent from "./components/ThingComponent";
 import OtherComponent from "./components/Other/OtherComponent";
 
 const App: React.FC = () => {
-  return <div></div>;
+  return (
```

Comment:
App, which renders either the `ThingComponent` or `OtherComponent` based on what has been selected from the navigation bar.